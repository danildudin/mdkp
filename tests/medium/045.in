28 9 0
710 106 626 848 636 474 415 829 6 898 705 895 394 262 237 506 359 126 73 394 862 861 636 464 724 240 306 425 
543 305 294 981 619 562 249 293 708 576 352 890 827 571 229 337 7 412 345 365 480 97 78 388 795 340 97 959 
858 610 149 140 413 452 619 857 166 832 250 825 91 765 477 181 206 764 419 349 962 518 681 996 43 567 409 585 
21 10 373 207 542 591 973 172 514 367 792 116 617 189 931 514 134 537 529 229 325 393 395 488 764 818 251 827 
248 285 852 93 449 346 852 923 345 919 797 802 976 76 35 855 636 172 536 251 842 328 515 302 883 688 883 119 
922 896 623 367 712 433 342 780 343 916 200 83 520 1 257 198 477 115 983 857 250 501 270 345 355 708 271 315 
580 771 345 14 555 428 847 498 277 246 789 357 113 496 990 108 821 114 267 724 433 372 460 967 635 125 627 661 
333 122 812 463 123 676 881 544 964 146 199 146 16 198 745 835 663 962 494 631 369 492 76 916 704 162 388 541 
785 902 45 881 395 203 38 50 59 560 691 451 402 569 809 568 487 598 69 860 247 35 329 735 128 19 828 601 
653 229 529 501 37 164 502 164 487 328 819 569 545 789 716 630 284 372 134 846 555 783 179 29 532 488 654 355 
7730 6002 6074 7116 9124 9635 7365 5632 8004 
