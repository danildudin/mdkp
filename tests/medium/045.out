8797
13
0 2 3 4 5 9 10 11 16 21 22 24 27 
